src/components/PropertyCard/PropertyCard.js

```diff
@@ -29,14 +77,18 @@ export default function PropertyCard({ property }) {
 				</CardContent>
 			</CardActionArea>
 
-			<CardActions disableSpacing>
-				<IconButton aria-label="add to favorites">
-					<FavoriteIcon />
+			<CardActions disableSpacing style={{ position: 'absolute', bottom: '0' }}>
+				<IconButton aria-label="add to favorites" onClick={handleLike}>
+					{like ? <FavoriteIcon fontSize="large" sx={{ color: 'red' }} /> : <FavoriteIcon fontSize="large" />}
 				</IconButton>
+
 				<IconButton aria-label="share">
 					<ShareIcon />
 				</IconButton>
 				<AvatarGroup max={4}>
+					{avatarUrls.map(url=>{
+						return <Avatar alt="Remy Sharp" src={url} key={url} />
+					})}
```

Comment:
if we have this, are the three hardcoded avatars below (lines 92-94) needed? if not, can they be deleted?

Reply:
also we need a task for getting this to actually work fr fr

Reply:
I left them here because the cards won't look so blank without much data. We can remove them anytime.

Reply:
This is actually working, we just don't have enough users and likes to fill up the space right now (if we don't want the placeholders).